airflow/bin/cli.py

```diff
@@ -28,6 +28,12 @@
 from airflow.utils.state import State
 from airflow.exceptions import AirflowException
 
+# make this configurable
+from airflow.api.client.local_client import LocalClient
+from airflow.api.client.api_client import ApiClient
+
+#client = LocalClient()
+client = ApiClient("http://localhost:8080")
```

Comment:
This assumes a web-server is running on the same machine as the CLI? it assumes a port? The port and host need to be in the config. However, that alone is not enough. This has operational implications for existing users that need to be thought through. I know of a few companies running on AWS that have security group-based access to the DB, but not to the webserver from nodes running the CLI.